### PR TITLE
Revamp article to monitor APM Usage

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Revamp article to monitor APM Usage

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 1]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
+### Set alerts on ingested bytes
 
-1. Go to Monitors -> New Monitor
-2. Set up a [new metric monitor][4] with `datadog.apm.host_instance`
-3. Define the rate you would like to set as a warning or error.
-4. Define an explicit notification: The volume of hosts on this env just got too high has exceeded the allocated threshold value. Scale down the number of APM enabled hosts.
+To ensure that your Ingested Spans usage remains within the allocation that APM Hosts and APM Fargate Tasks grant you, set up monitors to get alerted when your monthly usage gets close to your allocation.
 
-## Set alert on indexed spans
+1. Create a [metric monitor][8]
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:30]:
```suggestion
1. Create a [metric monitor][8].
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 2]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
+2. Define your query by using the `datadog.estimated_usage.apm.ingested_bytes` metric.
+3. Define the monitor's evaluation window to be `current month (MTD)`. This will ensure that the monitor is looking at the month-to-date usage. Read more about cumulative time windows in the [monitors][9] documentation.
+4. Define the Alert threshold and an optional Warning threshold to get alerted when the ingested volume reached 80 or 90% of your allotment.
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:33]:
```suggestion
4. Define the **Alert threshold** and an optional **Warning threshold** to alert when the ingested volume reaches 80% or 90% of your allotment. 
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 3]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
-3. Define the rate you would like to set as a warning or error.
-4. Define an explicit notification: The volume of hosts on this env just got too high has exceeded the allocated threshold value. Scale down the number of APM enabled hosts.
+To ensure that your Ingested Spans usage remains within the allocation that APM Hosts and APM Fargate Tasks grant you, set up monitors to get alerted when your monthly usage gets close to your allocation.
 
-## Set alert on indexed spans
+1. Create a [metric monitor][8]
+2. Define your query by using the `datadog.estimated_usage.apm.ingested_bytes` metric.
+3. Define the monitor's evaluation window to be `current month (MTD)`. This will ensure that the monitor is looking at the month-to-date usage. Read more about cumulative time windows in the [monitors][9] documentation.
+4. Define the Alert threshold and an optional Warning threshold to get alerted when the ingested volume reached 80 or 90% of your allotment. 
+5. Define the monitors name and notification policy to get alerted/alert your team when the ingested volumes are too high.
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:34]:
```suggestion
5. Enter a name for the monitor. Define the notification to send an alert to your team when the ingested volumes are too high.
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 4]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
-To get alerts in case a code deployment causes a spike in Indexed Spans generated, set up [Analytics monitors][5] on Indexed Spans. Get notified at any moment if the Indexed Span volumes in any infrastructure scope (for example,`service`, `availability-zone`) is growing unexpectedly:
+{{< img src="account_management/billing/monitor_usage_apm.png" alt="Monitor Usage APM" width="80%" >}}
 
-1. Go to [Analytics view][6] in APM
-2. Select the `env` (you can select `*`)
-3. Select `count` (you can select `*`)
-4. Select Export -> Export to Monitor
-5. Define the Indexed Span volume rate you would like to set as a warning or error.
-6. Define an explicit notification: The volume of Indexed Spans on this service just got too high. Define an additional exclusion filter or increase the filtering rate to put it back under control.
+To effectively reduce your ingested volumes, refer to this [guide][7] or to the ingestion mechanisms [documentation][10].
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:38]:
```suggestion
To effectively reduce your ingested volumes, see this [guide][7] or the [ingestion mechanisms][10] documentation.
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 5]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
 
-Learn more about [retention filters][7].
+### Set alerts on indexed spans
+
+Similarly, you can set alerts to ensure that your indexed spans budget remains within control. Create a metric monitor using the `datadog.estimated_usage.apm.indexed_spans` metric to get alerted when your month-to-date indexed spans volume goes over a defined threshold.
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:42]:
```suggestion
Similarly, you can set alerts to ensure that your budget for you indexed spans remains within certain limits. Create a metric monitor using the `datadog.estimated_usage.apm.indexed_spans` metric to get alerted when your month-to-date indexed spans volume goes over a defined threshold.
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 6]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
+
+To reduce the number of indexed spans, check your retention filters configuration. Read more about retention filters in the [documentation][11].
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:44]:
```suggestion
To reduce the number of indexed spans, check your configuration for retention filters. Read more about retention filters in the [trace retention][11] documentation.
```

File: content/en/account_management/billing/usage_monitor_apm.md [hunk 7]
```diff
@@ -8,42 +8,49 @@ Read APM documentation on [APM Billing][2] to understand how billing works for A
...
 
-## Set alert on indexed spans
+1. Create a [metric monitor][8]
+2. Define your query by using the `datadog.estimated_usage.apm.ingested_bytes` metric.
+3. Define the monitor's evaluation window to be `current month (MTD)`. This will ensure that the monitor is looking at the month-to-date usage. Read more about cumulative time windows in the [monitors][9] documentation.
+4. Define the Alert threshold and an optional Warning threshold to get alerted when the ingested volume reached 80 or 90% of your allotment. 
+5. Define the monitors name and notification policy to get alerted/alert your team when the ingested volumes are too high.
 
-To get alerts in case a code deployment causes a spike in Indexed Spans generated, set up [Analytics monitors][5] on Indexed Spans. Get notified at any moment if the Indexed Span volumes in any infrastructure scope (for example,`service`, `availability-zone`) is growing unexpectedly:
+{{< img src="account_management/billing/monitor_usage_apm.png" alt="Monitor Usage APM" width="80%" >}}
```
Comment (maycmlee) [inline content/en/account_management/billing/usage_monitor_apm.md:36]:
```suggestion
{{< img src="account_management/billing/monitor_usage_apm.png" alt="A metric monitor configuration page showing the datadog.estimated_usage.apm.ingested_bytes as the metric query" width="80%" >}}
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback

**Comment by dussault-antoine** [comment]
Thanks for the review ! Good to merge

**Review by maycmlee — APPROVED** [review]

**Pull request merged**